File: client/src/test/java/com/example/client/HelloClientTest.java
package com.example.client;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

class HelloClientTest {

  @Test
  void greetReturnsHelloWorld() {
    HelloClient client = new HelloClient();
    assertEquals("Hello world", client.greet());
  }
}

File: client/src/test/kotlin/com/example/client/HelloClientTest.kt
package com.example.client

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class HelloClientTest {
    @Test
    fun greetReturnsHelloWorld() {
        val client = HelloClient()
        assertEquals("Hello world", client.greet())
    }
}
